pallets/gear/src/weights.rs

```diff
@@ -44,11 +44,14 @@ pub trait WeightInfo {
     fn db_read_per_kb(c: u32, ) -> Weight;
     fn instantiate_module_per_kb(c: u32, ) -> Weight;
     fn claim_value() -> Weight;
```

Comment:
please rename old weights fn in favour of understanding of the case charged:
claim_value -> claim_value_non_zero_value etc